cognite/client/_api/datapoints.py

```diff
@@ -393,15 +400,27 @@ def retrieve_dataframe(
                 >>> from cognite.client import CogniteClient
                 >>> c = CogniteClient()
                 >>> df = c.datapoints.retrieve_dataframe(id=[1,2,3], start="2w-ago", end="now",
-                ...         aggregates=["average"], granularity="1h")
+                ...         aggregates=["average","sum"], granularity="1h")
+
+            Get a pandas dataframe with the index regularly spaced at 1 minute intervals, missing values completed and without the aggregate name in the columns::
```

Comment:
It is confusing to me as a reader that "missing values are completed" but at the same time I am passing a parameter "dropna".

Reply:
Maybe something like "missing values completed or dropped depending on the nature of the aggregate"